Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -519,16 +518,6 @@ public String getProcessingTimeAsFormattedString(Task task) {
         return Helper.getDateAsFormattedString(task.getProcessingTime());
     }
 
-    // a parameter is given here (even if not used) because jsf expects setter
-    // convention
-    public void setProcessingTimeNow(Task task) {
```

Comment:
You are sure to remove this getter and setter method as they referenced in `Kitodo/src/main/webapp/WEB-INF/templates/includes/tasks/taskBoxActivities.xhtml`?

Reply:
It uses getProcessingTimeNow() (it returns 1...) from Task bean. In general it would be better to remove also this occurrence from taskBoxActivities. Time now could be created just in background.